packages/docs/src/page-configs/ui-elements/data-table/examples/Other.vue

```diff
@@ -83,9 +90,10 @@
     :footer-clone="footerClone"
     :allow-footer-sorting="footerSorting"
     :animated="animated"
-    @row:click="rowEventType = $event.event.type, rowId = $event.item.id"
-    @row:dblclick="rowEventType = $event.event.type, rowId = $event.item.id"
-    @row:contextmenu="rowEventType = $event.event.type, rowId = $event.item.id"
+    :row-class="isTableRowClass && getCustomRowClass"
```

Comment:
I'm wondering if we can also implement `row-style` in scope of this PR. They're very similar.

See https://www.ag-grid.com/javascript-data-grid/row-styles/.

I would hope to get something like this:

```ts
type ClassDeclaration = string | string[] | Record<string, boolean>
rowClass = ClassDeclaration | () => ClassDeclaration

type StyleDeclaration = string | string[] | Record<string, string | number>
rowStyle = StyleDeclaration | () => StyleDeclaration
```

But it might be better idea to check vue types for classes and styles and just use these.

Reply:
Done